### ISSUE TITLE
Pop-ups and interactive or flashing content

### ISSUE THREAD

**Issue by Angelsaur** [alan-turing-institute/AutSPACEs#63]
### Summary

Pop-ups or flashing images can be very distracting. Users need to be able to disable or dismiss these elements easily.

### Deadline 

20/02/2021

### What needs to be done?

**Check that users can disable animated or moving content**
Things like scrolling carousels or blinking images can be very distracting for people with cognitive disabilities.

If your website contains these sorts of elements, you need to check whether users can disable them.

Go to any of your sample pages containing these items and see if there’s a way to stop them. Try clicking on them, or interacting with the surrounding content. Check to see if there’s a button to stop or pause the item.

**Check for content that plays automatically**
Some websites play audio or video content automatically when a user refreshes or opens a page.

If this applies to your website - and the content plays for more than 3 seconds - you’ll need to check users can either change the volume or stop it altogether. There’s likely to be a video or audio box somewhere on the page that you can interact with to do these things.

**Comment by jhlink** [comment]
Removing this from milestone 2. This should be lumped with the MVP - Accessibility ticket, or even better, an entire sprint could be dedicated to this.

**Comment by gedankenstuecke** [comment]
Made issue #369 to create an accessibility checklist that can be used for the MVP and going forward. Closing this along with other accessibility checklist items for now.
